src/dataExplorer/components/Results.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {FC, useState} from 'react'
+import {FlexBox, FlexDirection, ComponentStatus} from '@influxdata/clockface'
+
+import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
+import TimeZoneDropdown from 'src/shared/components/TimeZoneDropdown'
+
+import './Results.scss'
+
+// simplified version migrated from src/flows/pipes/Table/view.tsx
+const QueryStat: FC = () => {
+  const results: Record<string, any> = {}
```

Comment:
Is this just a place holder, no real results passed in right?

Reply:
Correct